homeassistant/components/velbus/manifest.json

```diff
@@ -13,7 +13,7 @@
     "velbus-packet",
     "velbus-protocol"
   ],
-  "requirements": ["velbus-aio==2023.11.0"],
+  "requirements": ["velbus-aio==2023.12.0"],
```

Comment:
Please bump in a separate PR

Reply:
^

Reply:
the bump will be required for the new code base to work..

Reply:
Yes, but the code isn't required for the bump to work. So I would love to see a different PR with the bump and then we can rebase this on the latest version and this keeps the PRs pure, small and clean.

Reply:
separate dependency bump was already accepted and merged: https://github.com/home-assistant/core/pull/107066

Reply:
@joostlek  - would you be able to review?